src/components/RowBetween/types.ts

```diff
@@ -0,0 +1,6 @@
+import {ReactNode} from 'react';
+import {ViewProps} from 'react-native';
+
+export interface RowBetweenProps extends ViewProps {
+    children?: ReactNode;
```

Comment:
```suggestion
    children?: React.ReactNode;
```
Add the React.ReactNode type instead of the destructured one

Reply:
and remove the unused import later